python/pyspark/ml/classification.py

```diff
@@ -64,10 +91,56 @@ def numClasses(self):
         return self._call_java("numClasses")
 
 
+class JavaProbabilisticClassifierParams(HasProbabilityCol, HasThresholds, JavaClassifierParams):
+    """
+    (Private) Java Probabilistic Classifier Params for classification tasks.
+    """
+    pass
```

Comment:
I'm OK with adding this class for parity, though indeed they are no-ops in Python. In Scala they only contain some internal logic. I'd be OK removing these too, but it's OK to have if it's not costing much of anything.

Reply:
@srowen Thanks for your comments.
I did a quick test using ```print (time.clock() - start_time, "seconds")``` in ```LogisticRegressionTest```to compare the difference between with and without this no-op class. I didn't see any time degradation with this no-op class. 
Also, I prefer to keep this no-op class so I don't need to make both ```JavaProbabilisticClassifier``` and ```JavaProbabilisticClassificationModel``` to implement ```HasProbabilityCol, HasThresholds```. I guess we just keep this class?

Reply:
This no-op class will help maintain the codebase if it's not costing much.